lib/blocks.js

```diff
@@ -352,6 +352,12 @@ BlockController.prototype.getBlockReward = function(height) {
     return 0;
   }
 
+  // If we've actived blossom, cut the reward in half
+  // https://github.com/zcash/zips/blob/master/zip-0208.rst
+  if (height > 653600) {
+    subsidy /= 2;
```

Comment:
I think this way of dividing by 2 would work as well as incrementing `halvings` as I proposed. I guess it's a matter of taste; I'm fine either way. (Also, of course, this would have to be conditional on testnet versus mainnet.)